### PR TITLE
NDRCN-324 Testing Fixes

### PR DESCRIPTION
**Pull request by somaraja**

### CODE REVIEW

File: app/views/BulkFileUploadView.scala.html [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
     @govukFileUpload(FUpload(
         name = "file",
-        id = "upload-file",
```
Comment (anu9) [inline app/views/BulkFileUploadView.scala.html:76]: Does this affect error focus?

Comment (somaraja) [reply app/views/BulkFileUploadView.scala.html:76]: Yes, fixed this issue

Comment (anu9) [reply app/views/BulkFileUploadView.scala.html:76]: lgtm

Comment (anu9) [reply app/views/BulkFileUploadView.scala.html:76]: lgtm

File: app/controllers/BulkFileUploadController.scala [hunk 2]
```diff
@@ -54,7 +54,8 @@ class BulkFileUploadController @Inject()(
                                         )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport with FileUploadService {
 
   final val COOKIE_JSENABLED = "jsenabled"
```
Comment (chandaksweta) [inline app/controllers/BulkFileUploadController.scala:56]: this is no longer needed

File: app/controllers/BulkFileUploadController.scala [hunk 3]
```diff
@@ -54,7 +54,8 @@ class BulkFileUploadController @Inject()(
...
-  final val controller = routes.FileUploadController
+  final val fileUploadController = routes.FileUploadController
+  final val bulkFileUploadController = routes.BulkFileUploadController
   type ConvertState = (FileUploadState) => Future[FileUploadState]
```
Comment (chandaksweta) [inline app/controllers/BulkFileUploadController.scala:59]: this is not needed too

File: app/controllers/BulkFileUploadController.scala [hunk 4]
```diff
@@ -96,35 +109,54 @@ class BulkFileUploadController @Inject()(
...
   private def updateSession(newState: FileUploadState, userAnswers: Option[UserAnswers]) = {
     if (userAnswers.nonEmpty)
       sessionRepository.set(userAnswers = userAnswers.get.copy(fileUploadState = Some(newState)))
     else Future.successful(true)
   }
 
-  final def successRedirect(id: String, userAnswers: UserAnswers)(implicit rh: RequestHeader) = appConfig.baseExternalCallbackUrl + (rh.cookies.get(COOKIE_JSENABLED) match {
-    case Some(_) => getBulkEntryDetails(userAnswers)
-    case None => controller.showWaitingForFileVerification
+  final def successRedirect(id: String)(implicit rh: RequestHeader) = appConfig.baseExternalCallbackUrl + (rh.cookies.get(COOKIE_JSENABLED) match {
```
Comment (chandaksweta) [inline app/controllers/BulkFileUploadController.scala:137]: This needs refactoring. Get rid of this method and in the callback add the url directly

File: app/controllers/BulkFileUploadController.scala [hunk 5]
```diff
@@ -96,35 +109,54 @@ class BulkFileUploadController @Inject()(
...
+    case Some(_) => bulkFileUploadController.showWaitingForFileVerification
+    case None => bulkFileUploadController.showWaitingForFileVerification
   })
 
   final def errorRedirect(id: String)(implicit rh: RequestHeader) =
```
Comment (chandaksweta) [inline app/controllers/BulkFileUploadController.scala:142]: same here

File: conf/app.routes [hunk 6]
```diff
@@ -214,6 +214,8 @@ POST       /change-indirect-representative                  controllers.Indirect
 GET         /upload-proof-of-authority                       controllers.ProofOfAuthorityController.onPageLoad()
 
 GET        /upload-multiple-entries                        controllers.BulkFileUploadController.showFileUpload()
```
Comment (chandaksweta) [inline conf/app.routes:216]: the urls for bulk looks mis aligned.

**Review by chandaksweta — CHANGES_REQUESTED** [review]
Overall looks good to me. The refactoring needs to be done and the urls should be in sync

File: app/controllers/BulkFileUploadController.scala [hunk 7]
```diff
@@ -68,7 +69,7 @@ class BulkFileUploadController @Inject()(
           (checkStateActor ? CheckState(request.internalId, LocalDateTime.now.plusSeconds(30), s)).mapTo[FileUploadState].flatMap {
             case s: FileUploaded => {
               logger.info(s"File uploaded $s")
```
Comment (chandaksweta) [inline app/controllers/BulkFileUploadController.scala:71]: remove logging.. no longer needed

**Review by chandaksweta — APPROVED** [review]

**Pull request merged**